repaygithub/ui-tools
# Javascript unit test

How to test:
- Generate an app, ts or js
-Run yarn test at the root and check that everything works as expected
